File: AUGUST-2025/05-08-2025/p3477.rb
# @param {Integer[]} fruits
# @param {Integer[]} baskets
# @return {Integer}
def num_of_unplaced_fruits(fruits, baskets)
    fruits.size - fruits.count {|f|
        baskets.size.times.any? {|i| ((baskets[i] >= f) ? (baskets.delete_at(i); true) : false) }
    }
end

File: AUGUST-2025/05-08-2025/p3477.js
/**
 * @param {number[]} fruits
 * @param {number[]} baskets
 * @return {number}
 */
var numOfUnplacedFruits = function (fruits, baskets) {
    let count = 0
    for (let i = 0; i < fruits.length; i++) {
        for (let j = 0; j < baskets.length; j++) {
            if (fruits[i] <= baskets[j]) {
                baskets[j] = 0
                count++
                break
            }
        }
    }
    let res = fruits.length - count
    return res
};